### PR TITLE
open-mpi 5.0.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/o/open-mpi.rb [hunk 1]
```diff
@@ -39,10 +39,6 @@ def install
       ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
     end
 
-    # Work around asm incompatibility with new linker (FB13194320)
-    # https://github.com/open-mpi/ompi/issues/11935
-    ENV.append "LDFLAGS", "-Wl,-ld_classic" if DevelopmentTools.clang_build_version >= 1500
-
```
Comment (chenrui333) [inline Formula/o/open-mpi.rb:45]: resolved per https://github.com/open-mpi/ompi/issues/11935

**Comment by chenrui333** [comment]
```
  open-mpi
    * Files were found with references to the Homebrew shims directory.
      The offending files are:
        bin/prte_info
        bin/pmix_info
```

**Comment by chenrui333** [comment]
rerun

**Comment by chenrui333** [comment]
```
  checking for pmix static libs... -lpmix
  checking for pmix.h... no
  configure: error: Could not find viable pmix build.
```

**Comment by chenrui333** [comment]
> ```
>   checking for pmix static libs... -lpmix
>   checking for pmix.h... no
>   configure: error: Could not find viable pmix build.
> ```

see #152875

**Comment by SMillerDev** [comment]
Needs bumps for: cp2k opencoarrays petsc petsc-complex vineyard

**Comment by chenrui333** [comment]
rerunning 🤞

**Review by SMillerDev — APPROVED** [review]

**Comment by chenrui333** [comment]
just `glslviewer` linux regression build failure, I will followup on it.

**Pull request merged**